content/en/docs/instrumentation/python/_index.md

```diff
@@ -13,19 +13,78 @@ the generation and collection of application telemetry data such as metrics,
 logs, and traces. This documentation is designed to help you understand how to
 get started using OpenTelemetry for Python.
 
+## Version support
+
+OpenTelemetry-Python supports Python 3.6 and higher.
+
 ## Status and Releases
 
 The current status of the major functional components for OpenTelemetry Python
 is as follows:
 
-| Tracing   | Metrics | Logging      |
-| -------   | ------- | ------------ |
-| Stable    | Alpha   | Experimental |
+- **Tracing** ([API][api/t], [SDK][sdk/t]): [Stable][]
```

Comment:
What's the difference between alpha and experimental?

Reply:
I think experimental for both is fine for Python

Reply:
From the spec (https://opentelemetry.io/docs/reference/specification/versioning-and-stability/#experimental, emphasis mine):

> Signals start as experimental, which covers **alpha**, beta, and release candidate versions of the signal.

Reply:
I think alpha is fine because it's technically true, right? The API is implemented and there's SDK support but it's not final? @codeboten check my work here